src/partials/products.html

```diff
@@ -1,65 +1,57 @@
-    <section class="products section">
-      <div class="container products__container">
-        <div class="products__title">
-          <label id="natural">100% NATURAL</label>
-          PRODUCTS
-        </div>
-        <ul class="products__list">
-          <li>
-            <figure class="products__card_1">
-              <img
-                class="product__image_1"
-                src="images/product-1.png"
-                alt="ICE CREAM"
-                width="335"
-                height="335"
-              />
-              <figcaption class="products__text-flex">
-                <h2 class="products__name">ICE CREAM</h2>
-                <span class="products__description"
-                  >Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-                  do eiusmod tempor incididunt ut labore et</span
-                >
-              </figcaption>
-            </figure>
-          </li>
-          <li>
-            <figure class="products__card_2">
-              <img
-                class="product__image_2"
-                src="src/images/product-2.png"
-                alt="ICE COFFEE"
-                width="335"
-                height="335"
-              />
-              <figcaption class="products__text-flex">
-                <h2 class="products__name">ICE COFFEE ...</h2>
-                <span class="products__description"
-                  >Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-                  do eiusmod tempor incididunt ut labore et</span
-                >
-              </figcaption>
-            </figure>
-          </li>
-          <li>
-            <figure class="products__card_3">
-              <img
-                class="product__image_3"
-                src="images/product-3.png"
-                alt="MILKSHAKES"
-                width="335"
-                height="335"
-              />
-              <figcaption class="products__text-flex">
-                <h2 class="products__name">MILKSHAKES ...</h2>
-                <span class="products__description"
-                  >Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-                  do eiusmod tempor incididunt ut labore et</span
-                >
-              </figcaption>
-            </figure>
-          </li>
-        </ul>
-      </div>
-    </section>
+<link rel="stylesheet" href="../sass/products/products.scss" />
```

Comment:
to podgladu pracy spoko, ale pamietaj to potem usunąć, bo będzie jeden plik css wygenerowany przez sass do calego projektu